includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -128,8 +128,10 @@ public function __construct() {
 		// When feature flags are enabled, title shows the count of enabled payment methods in settings page only.
 		if ( WC_Stripe_Feature_Flags::is_upe_checkout_enabled() && WC_Stripe_Feature_Flags::is_upe_preview_enabled() && isset( $_GET['page'] ) && 'wc-settings' === $_GET['page'] ) {
 			$enabled_payment_methods_count = count( $this->get_upe_enabled_payment_method_ids() );
-			/* translators: $1. Count of enabled payment methods. */
-			$this->title = sprintf( _n( '%d payment method', '%d payment methods', $enabled_payment_methods_count, 'woocommerce-gateway-stripe' ), $enabled_payment_methods_count );
+			$this->title                   = $enabled_payment_methods_count ?
+				/* translators: $1. Count of enabled payment methods. */
+				sprintf( _n( '%d payment method', '%d payment methods', $enabled_payment_methods_count, 'woocommerce-gateway-stripe' ), $enabled_payment_methods_count )
+				: $this->method_title;
```

Comment:
Need to set the `title` same as the `method_title` when `$enabled_payment_methods_count` is `0`.
If `title` and `method_title` are different, a dash (`-`) appears beside the method name even when the title is empty.
That logic is coming from [this code](https://github.com/woocommerce/woocommerce/blob/trunk/includes/admin/settings/class-wc-settings-payment-gateways.php#L170).

Reply:
Good find!